assembl/static2/css/components/posts.scss

```diff
@@ -141,7 +141,7 @@
 
     .body {
         padding: 20px 0;
-        word-break: break-all;
+        word-wrap: break-word;
```

Comment:
Maybe replace `word-wrap` occurrences here by `overflow-wrap` because it seems to be the real official correct name of the property according to https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap